registry/storage/manifeststore.go

```diff
@@ -98,7 +101,9 @@ func (ms *manifestStore) Get(ctx context.Context, dgst digest.Digest, options ..
 		switch versioned.MediaType {
 		case schema2.MediaTypeManifest:
 			return ms.schema2Handler.Unmarshal(ctx, dgst, content)
-		case manifestlist.MediaTypeManifestList:
+		case ocischema.MediaTypeManifest:
+			return ms.ocischemaHandler.Unmarshal(ctx, dgst, content)
```

Comment:
kk added a "WIP" handler for ocischema manifests if you want to test it out... but I need to go over the delta's between ocischema and schema2 to see if anything special needs to be stored other than the annotations I haven't gotten to yet :-)